src/components/UI/docs/MDComponents.tsx

```diff
@@ -92,11 +92,9 @@ const MDComponents = {
     );
   },
   h4: ({ children }: any) => {
-    const { children: parsedChildren, headingId } = parseHeadingId(children);
-
     return (
-      <Heading as='h4' mb='2.5 !important' {...h4} id={headingId}>
-        {parsedChildren}
+      <Heading as='h4' mb='2.5 !important' {...h4} id={children}>
```

Comment:
Hmm what if the heading content contains a whitespace? will be an invalid html id. Or is that case not possible here?

Reply:
Seeing this in the preview
![image](https://github.com/user-attachments/assets/eeee2291-fc85-417f-b1f9-e8747f3b0fc9)